product_docs/docs/pgd/5/node_management/node_types.mdx

```diff
@@ -4,19 +4,17 @@ title: Node types
 
 ## Data nodes
 
-A data node in PGD is a node which runs a Postgres instance. It replicates data to all other data nodes. It also participates in the cluster wide Raft decision making around locking and leadership. It will be the member of one or more groups and by default will be a member of the "top level" group which spans all data nodes in the cluster.
+A data node in PGD is a node that runs a Postgres instance. It replicates data to all other data nodes. It also participates in the cluster-wide Raft decision-making around locking and leadership. It will be the member of one or more groups and by default will be a member of the "top level" group that spans all data nodes in the cluster.
```

Comment:
I had a query here about the use of future tense here. This is creating a question for me about when this happens. Could you clarify or recast without future tense?

Reply:
I believe this would work in the present tense as well. 
```suggestion
A data node in PGD is a node that runs a Postgres instance. It replicates data to all other data nodes. It also participates in the cluster-wide Raft decision-making around locking and leadership. It can be a member of one or more groups and is, by default, a member of the "top level" group that spans all data nodes in the cluster.
```